tests/components/tibber/test_services.py

```diff
@@ -110,96 +106,60 @@ def generate_mock_home_data():
     return mock_homes
```

Comment:
We don't need to set name again on the lines above.

Reply:
Looks like we actually do, as `name` is a special property of `MagicMock` objects.
Without setting it again, the tests fail.
`mock_home.name` will be another `MagicMock` object instead of the `str` we need it to be.

See documentation: https://docs.python.org/3/library/unittest.mock.html#attaching-mocks-as-attributes

Reply:
Right, thanks.